src/pages/patch/patchTabs/DownstreamTasks.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Table } from "antd";
+import { Patch } from "gql/generated/types";
+import { DownstreamProjectAccordion } from "./DownstreamProjectAccordion";
+
+type childPatchesType = Patch["childPatches"];
+type childPatch = childPatchesType[0];
+
+interface DownstreamTasksProps {
+  childPatches: childPatchesType;
+}
+
+export const DownstreamTasks: React.FC<DownstreamTasksProps> = ({
+  childPatches,
+}) => {
+  const columns = [
+    {
+      title: <span data-cy="child-patch">Child Patch</span>,
+      render: (
+        text: string,
+        { project, patchID, status, taskCount }: childPatch
+      ): JSX.Element => (
+        <DownstreamProjectAccordion
+          key={`downstream_project_${patchID}`}
+          projectName={project}
+          status={status}
+          childPatchId={patchID}
+          taskCount={taskCount}
+        />
+      ),
+    },
+  ];
+
+  return (
+    <Table
+      tableLayout="fixed"
+      data-cy="downstream-tasks-table"
+      dataSource={childPatches}
```

Comment:
I think if we are removing all the table styling and not actually using any of the table features we should just omit it and just render out the `<DownstreamProjectAccordion />` with out it. No need to add any of the additional table weight if we don't need it.